enhancements/single-node/aos-sno-pair.md

```diff
@@ -0,0 +1,143 @@
+---
+title: aos-sno-pair
+authors:
+- @mshitrit
+  reviewers:
+- TBD
+  approvers:
+- TBD
+
+creation-date: 2021-07-12
+last-updated: 2021-07-12
+status: implementable
+---
+
+# Atomic OpenShift Single-Node-Pair
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+This enhancement describes setting up a two nodes cluster with a reduced footprint which will serve edge cases.
```

Comment:
```suggestion
This enhancement does **not** propose replacing or otherwise modifying OpenShift's use of etcd.

This enhancement proposes a layer on top of exactly two standard Single Node OpenShift clusters that monitors for node-level failures and uses existing OpenShift APIs to scale up Deployments once the peer has been put into a safe state.
```